### DISCUSSION TITLE
LiteSpeed / OpenLiteSpeed compatibility

### DISCUSSION THREAD

**Discussion by TOMCO984** [openappsec/openappsec#235] [Feature Requests]
Hi,
This can be a good to add compatibility with LiteSpeed and OpenLiteSpeed.

Have a nice day.

**Comment by Fieldmouse88** [comment]
Is this possible? Can I use open-appsec with my litespeed web server somehow?

**Reply by orianelou** [reply]
Hi @Fieldmouse88,

Currently not, but we're collecting interest for this

**Reply by Fieldmouse88** [reply]
I happen to have an nginx proxy faceing internet and reverse proxying to the litespeed web server, would it work to add open-appsec to the nginx reverse proxy?

**Comment by orianelou** [comment]
If you're interested in this feature, please 👍 or upvote this issue to show your support!

**Comment by J-Rey** [comment]
Well, it seems easiest to roll out support for their K8s Ingress Controller to start with but that only works with their licensed Web ADC load balancer. Their regular enterprise & open source web servers are more popular though and have an API, but they all are compatible with Apache HTTP Server configs if that one's also on the roadmap. So a few options to consider.
